galata/test/jupyterlab/workspace.test.ts

```diff
@@ -228,6 +228,28 @@ test.describe('Workspace', () => {
     await expect(page.locator(`[role="main"] >> text=${mdFile}`)).toBeVisible();
     await expect(page.locator(`[role="main"] >> text=${nbFile}`)).toBeVisible();
   });
+
+  test('should display workspace indicator and switch workspaces', async ({
+    page
+  }) => {
+    await page.goto();
+    const workspaceSelector = page.locator('.jp-WorkspaceSelector-header');
+    //Not visible by default
```

Comment:
It's a bit easier to read with a space and the rest of codebase uses such style:

```suggestion
    // Not visible by default
```